Basic-Car-Maintenance/Shared/Dashboard/Views/DashboardView.swift

```diff
@@ -9,12 +9,12 @@ import SwiftUI
 
 struct DashboardView: View {
     
-    @StateObject private var viewModel: DashboardViewModel
-    @State private var isShowingEditView = false
+    @Bindable private var viewModel: DashboardViewModel
+    @State private var isShowingAddView = false
```

Comment:
I think this was merged wrong, there should be `isShowingEditView`, for editing (but I think an extra property also got left, so I'm going to fix it and merge this